deploy/charts/cert-manager/templates/deployment.yaml

```diff
@@ -32,8 +32,8 @@ spec:
 {{- if .Values.podLabels }}
 {{ toYaml .Values.podLabels | indent 8 }}
 {{- end }}
-      annotations:
       {{- if .Values.podAnnotations }}
+      annotations:
 {{ toYaml .Values.podAnnotations | indent 8 }}
```

Comment:
This change will cause an issue if prometheus/monitoring is enabled from what I can see:

```
      {{- if .Values.podAnnotations }}
      annotations:
{{ toYaml .Values.podAnnotations | indent 8 }}
      {{- end }}
      {{- if and .Values.prometheus.enabled (not .Values.prometheus.servicemonitor.enabled) }}
        prometheus.io/path: "/metrics"
        prometheus.io/scrape: 'true'
        prometheus.io/port: '9402'
      {{- end }}
```

The prometheus annotations will be added without a corresponding `annotations:` line

Reply:
Nice catch!  Let me know if this works for the project